XamlControlsGallery/.editorconfig

```diff
@@ -0,0 +1,109 @@
+﻿# EditorConfig  https://EditorConfig.org
```

Comment:
Is there a editor config property regarding point 4 of [this document](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md):

> We avoid `this.` unless absolutely necessary.

Reply:
Yes, however for right now I just wanted to rely on the minimal EditorConfig which includes only the rules that both the Roslyn compiler and .NET Core runtime teams agree on.

They don't agree on how to enforce the use of `.this`, so it isn't in the config for now. I added that part to the best practices before I did the study comparing both configs and making the minimal config.

I plan to work through the rules in this config for now, and add additional ones over time - such as addressing this. But I feel commits adding new (potentially controversial) rules should have their own PRs.

See [this](https://styleascode.net/EditorConfigs/selection.html) regarding the config options I went though, and [this](https://styleascode.net/EditorConfigs/style_as_code.html) for the configs I plan to make available over time.

Reply:
Hmm I see. I guess in that case, it is best to leave the `this.` controversy out (though I think in WinUI this is rarely used, if used at all).